web/app/scripts/resources/query-builder-service.js

```diff
@@ -25,11 +29,12 @@
          * @param {number} offset The page in django's pagination to return
          * @param {bool} doFilter If true: filter results
```

Comment:
The docstring should be updated; I'd also probably move the `extraParams` to be the last in the function's parameter list.